src/actions/actions.js

```diff
@@ -10,26 +10,22 @@ export const getTeams = () => async dispatch => {
     dispatch({ type: 'GET_TEAM_COLOR', payload: raptorsTeam.primaryColor });
 
     const raptorsRosterResponse = await dataNbaNet.get('/json/cms/noseason/team/raptors/roster.json');
-    dispatch({ type: 'GET_PLAYERS', payload: raptorsRosterResponse.data.sports_content.roster.players.player });
+    const raptorsRoster = raptorsRosterResponse.data.sports_content.roster.players.player.map((player) => {
+        return { ...player, teamColor: raptorsTeam.primaryColor };
+    });
+    dispatch({ type: 'GET_PLAYERS', payload: raptorsRoster });
 };
 
 export const getSelectedTeam = team => async dispatch => {
     dispatch({ type: 'GET_SELECTED_TEAM', payload: team });
     dispatch({ type: 'GET_TEAM_COLOR', payload: team.primaryColor });
 
-    const teamUrlName = () => {
-        if (team.ttsName === 'Philadelphia 76ers') {
-            return 'sixers';
-        }
-        return team.ttsName.trim().split(' ').pop().toLowerCase();
-    };
-    const teamRosterResponse = await dataNbaNet.get(`/json/cms/noseason/team/${teamUrlName()}/roster.json`);
-    dispatch({ type: 'GET_PLAYERS', payload: teamRosterResponse.data.sports_content.roster.players.player });
-};
-
-export const getPlayers = teamName => async dispatch => {
-    const response = await dataNbaNet.get(`/json/cms/noseason/team/${teamName}/roster.json`);
-    dispatch({ type: 'GET_PLAYERS', payload: response.data.sports_content.roster.players.player });
+    const teamUrlName = (team.teamId === '1610612755' ? 'sixers' : team.ttsName.trim().split(' ').pop().toLowerCase());
```

Comment:
Better to have a discussion about enums. The short of it is that instead of writing strings throughout our app, it's better to put these strings in an object and reference them.

This would be better discussed over a hangout or in person.